dist/src/check.js

```diff
@@ -23,16 +23,18 @@ function checkConfig(config) {
         console.error("No relaychain nodes defined");
         return false;
     }
-    if (config.relaychain.flags && config.relaychain.flags.constructor !== Array) {
-        console.error("Parachain flags should be an array.");
-        return false;
+    for (const node of config.relaychain.nodes) {
+        if (node.flags && node.flags.constructor !== Array) {
+            console.error("Parachain flags should be an array.");
+            return false;
+        }
     }
     if (!config.parachains) {
         console.error("Missing `parachains` object");
         return false;
     }
-    if (config.parachains.length >= config.relaychain.nodes.length) {
-        console.error("Must have more relaychain nodes than parachains.");
+    if (config.parachains.length > config.relaychain.nodes.length) {
+        console.error("Must have the same or greater number of relaychain nodes than parachains.");
```

Comment:
The logic changed one way and the comment changed the other way

Reply:
haha yes indeed. So which one is correct? Last time we discussed this a few month ago it was >=, no?

Reply:
I believe it is `>=`.